emsdk

```diff
@@ -1680,15 +1700,51 @@ def find_latest_releases_sdk(which):
   return 'sdk-releases-%s-%s-64bit' % (which, ident)
 
 
+def find_tot_sdk(which):
+  if not os.path.exists(tot_path()):
+    print('Tip-of-tree information was not found, run emsdk update-tags')
+    sys.exit(1)
+  tot = open(tot_path()).read()
+  if not tot:
+    print('Tip-of-tree build was not found, run emsdk update-tags and/or wait')
```

Comment:
I'm not really sure what this means, so it's probably even more obscure to a user.